src/cli/commands/workspaces.js

```diff
@@ -27,8 +29,30 @@ export async function info(config: Config, reporter: Reporter, flags: Object, ar
   const publicData = {};
 
   for (const workspaceName of Object.keys(workspaces)) {
+    const {loc, manifest} = workspaces[workspaceName];
+
+    const workspaceDependencies = new Set();
+    const mismatchedWorkspaceDependencies = new Set();
+
+    for (const dependencyType of DEPENDENCY_TYPES) {
+      if (dependencyType !== 'peerDependencies') {
+        for (const dependencyName of Object.keys(manifest[dependencyType] || {})) {
+          if (Object.prototype.hasOwnProperty.call(workspaces, dependencyName)) {
+            const request = manifest[dependencyType][dependencyName];
```

Comment:
`expectedRange`, `requestedRange`? Something that wouldn't confuse me with an actual HTTP request or the `request` module?